dist/rpm/os-autoinst-scripts-deps.spec

```diff
@@ -36,6 +38,11 @@ Suggests:       postgresql
 - auto-review - Automatically detect known issues in openQA jobs, label openQA jobs with ticket references and optionally retrigger
 - openqa-investigate - Automatic investigation jobs with failure analysis in openQA
 
+%package devel
+Summary:        Development package for os-autoinst-scripts-deps
+Group:          Development/Tools/Other
+Requires:       %devel_requires
+
```

Comment:
I'm not sure if this enough to create a new package in OBS

Reply:
Good point. I believe so. This should create a new subpackage, like in openQA, but you are likely missing a [`%description`](https://github.com/os-autoinst/openQA/blob/6ca685738c5db7ccdd9394dec8d4841a293c65ec/dist/rpm/openQA.spec#L170) and [`%files`](https://github.com/os-autoinst/openQA/blob/6ca685738c5db7ccdd9394dec8d4841a293c65ec/dist/rpm/openQA.spec#L642) sections.

Reply:
ah right, the description.
I added both now. `%files` is just empty, but it's probably needed.

Reply:
yes, an empty section is needed. Current version looks good now